db/app.js

```diff
@@ -0,0 +1,9 @@
+const endpoints = require('../endpoints.json');
+const express = require("express")
+const app = express()
+
+app.get('/api', (req, res) => {
+    res.status(200).send({ endpoints });
+});
```

Comment:
This is a good start but how would you refactor this to follow the MVC structure?


Notes:
https://l2c.northcoders.com/courses/sd-notes/back-end#sectionId=mvc,step=intro